docs/user-guide/installation/windows.md

```diff
@@ -18,18 +18,18 @@ _Note_: Keep in mind that your CMake version should be newer than the boost vers
 
 To set up your environment:
 
-1.  Install Microsoft Visual C++ 2015 or newer (2017 or later is recommended)[MSVC](https://visualstudio.microsoft.com/)
+1.  Install Microsoft Visual C++ 2017 or newer (2019 or later is recommended)[MSVC](https://visualstudio.microsoft.com/)
 2.  Install
-    [Boost](https://www.boost.org/doc/libs/1_70_0/more/getting_started/windows.html)
-    [Windows downloads](https://dl.bintray.com/boostorg/release/1.70.0/binaries/)
-    1.61 or later recommended (core library should build with 1.58,
-    but tests will not). For CMake to detect it automatically either
+    [Boost](https://www.boost.org/doc/libs/1_74_0/more/getting_started/windows.html)
+    [Windows downloads](https://dl.bintray.com/boostorg/release/1.74.0/binaries/)
+    1.67 or later recommended (core library should build with 1.58,
+    but tests and web interface will not). For CMake to detect it automatically either
     extract Boost to the root of your drive, or set the `BOOST_INSTALL_PATH`
     environment variable to the install location. The CMake will only automatically find
     boost 1.58 or newer.
-    Building with Visual Studio 2017 will require boost 1.65.1 or newer and CMake 3.9
-    or newer. Use 14.0 versions for Visual Studio 2015, 14.1 files for Visual studio 2017. Visual studio 2019 will require CMake 3.14 or later.
-    Boost 1.70 with CMake 3.14+ is the current recommended configuration.
+    Building with Visual Studio 2017 will require boost 1.67 or newer and CMake 3.15+
```

Comment:
```suggestion
    Building with Visual Studio 2017 will require boost 1.67 or newer and CMake 3.10+
```

Earlier it says CMake 3.10+ is required -- either this or the earlier line should be updated.